styx-flyte-client/src/main/proto/update_protos.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+FLYTEIDL_VERSION="0.17.7"
```

Comment:
`set -euo pipefail`

Reply:
Also it is better to use env to discover bash `#!/usr/bin/env bash`

Reply:
You can try to run shellcheck on this and it is usually pretty useful. Also we can consider adding it to circle ci in a later PR: https://circleci.com/orbs/registry/orb/circleci/shellcheck

Reply:
Ah nice to check with shellcheck.
It returned nothing after the recent additions 👍

Reply:
Cool